src/utils.py

```diff
@@ -16,41 +16,34 @@ def batch_iterator(
         yield dataset[i : i + batch_size][key]
 
 
-def find_files(data_dir, data_files):
-    if data_dir is None and data_files is None:
-        raise ValueError("At least one of data_files or data_dir must be specified")
-    elif data_dir is not None and data_files is None:
-        pathname = data_dir + "/**/*.*"
-        data_files = glob.glob(pathname, recursive=True)
-        data_files = [
-            p for p in data_files if p.endswith(".txt") or p.endswith(".json")
+def load_corpora(dir_path, text_type_per_line="docu"):
+    corpora_dir = Path(dir_path).absolute()
+    all_file_paths = []
+
+    if text_type_per_line == "docu":
+        list_of_file_paths = [
+            str(file_path) for file_path in corpora_dir.rglob("*.json")
         ]
-    elif data_dir is not None and data_files is not None:
-        if isinstance(data_files, str):
-            data_files = [data_files]
-        data_files = [os.path.join(data_dir, d) for d in data_files]
-    return data_files
 
+        if not list_of_file_paths:
+            raise Exception("source files must have 'json' extension.")
 
-def load_corpora(data_dir, cache_dir=None, split="train", data_files=None):
-    data_files = find_files(data_dir, data_files)
-    if data_files[0].endswith(".txt"):
-        return load_dataset(
-            "text",
-            data_dir=data_dir,
-            data_files=data_files,
-            cache_dir=cache_dir,
-            split=split,
-        )
-    elif data_files[0].endswith(".json"):
+        all_file_paths.extend(list_of_file_paths)
+        return load_dataset("json", data_files=all_file_paths, split="train")
+    elif text_type_per_line == "sent":
+        # sent
+        list_of_file_paths = [
+            str(file_path) for file_path in corpora_dir.rglob("*.txt")
+        ]
+        if not list_of_file_paths:
+            raise Exception("source files must have 'txt' extension.")
+
+        all_file_paths.extend(list_of_file_paths)
         return load_dataset(
-            "json",
-            data_dir=data_dir,
-            data_files=data_files,
-            cache_dir=cache_dir,
-            split=split,
+            SENTENCE_PER_LINE_SCRIPT, data_files=all_file_paths, split="train"
         )
-    raise ValueError("File formats other than '.txt' and '.json' are not supported yet")
+    else:
+        raise NotImplementedError("Implementing loading scripts along your text type.")
```

Comment:
```suggestion
        raise NotImplementedError(f"Implementing loading scripts along with your text type (docu, sent), but got {text_type_per_line}")
```
raise 시점에 유저 인풋이 무엇이었는지 보여주면 좋을 것 같습니다~